Week1/homework/app.js

```diff
@@ -1,11 +1,92 @@
+// assert = chai.assert.deepEqual
+
 'use strict';
+/*//make an array
+
+const bookList = [
+  'the-kite-runner',
+  'number-the-stars',
+  'pride-and-prejudice',
+  'the-outsiders',
+  'little-women',
+];
+//this is how I printed my array in the DOM but not like a list. like a line!
+document.querySelector('#myArr').innerHTML = `My bookList Array : ${bookList}`
+
+//Now I want to creat an unordered list out of the bookList array
+
+function createList() {
+  let uList = document.createElement('ul');
+  uList.setAttribute('id', 'li');
+  document.body.appendChild(uList);
+
+
+  for (let index in bookList) {
+    let eachBook = document.createElement('li');
+    uList.appendChild(eachBook);
+
+    eachBook.textContent = bookList[index];
+  }
+}
+createList();*/
 
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
-  ];
+//Now I have an object array of my must read books information
+
+
+
+let books = [
+  {
+    title: 'The Kite Runner',
+    author: 'Khaled Hosseini',
+    language: 'English',
+    cover: 'kite.jpg'
+  },
+  {
+    title: 'Number the Stars',
+    author: 'lois Lowry',
+    language: 'English',
+    cover: 'number.jpg'
+
+  },
+  {
+    title: 'Pride and Prejudice',
+    author: 'Jane Austen',
+    language: 'English',
+    cover: 'pride.jpg'
+  },
+  {
+    title: 'The Outsiders',
+    author: 'S.E Hinton',
+    language: 'English',
+    cover: 'outsiders.jpeg'
+
+  },
+  {
+    title: 'Little Women',
+    author: 'Louisa May',
+    language: 'English',
+    cover: 'Little.png'
+  }
+]
+
+document.body.onload = printBooks;
+//now I want to print all the information in the DOM.
+
+function createAndAppend(typ, parent, attributes = {}) {
```

Comment:
Ah, looks like this helper function makes a surprise appearance from the JS3 homework :)

Reply:
Just make sure you know what it is doing though 👍

Reply:
Yeah , Actually I did it in a different way , but in the session with Christer we changed it to look like this. I think I got how it works. :)